napari/utils/events/_tests/test_event_emitter.py

```diff
@@ -17,3 +17,36 @@ def test_event_blocker_count():
         e()
         e()
     assert block.count == 3
+
+
+def test_error_on_connect():
```

Comment:
```suggestion
def test_error_on_connect():
    """Check that connections happen correctly even on decorated methods.

    Some decorators will alter method.__name__, so that obj.method
    will not be equal to getattr(obj, obj.method.__name__). We check here
    that event binding will be correct even in these situations.
    """
```